manifests/extension.pp

```diff
@@ -43,6 +47,7 @@
   $compiler_packages = $::php::params::compiler_packages,
   $zend              = false,
   $settings          = {},
+  $settings_prefix   = true,
```

Comment:
I think we should default to false, at least until we release a new major version, due to the fact that not all extensions use their name as prefix (APCu uses `apc.`)